graphium/data/utils.py

```diff
@@ -105,3 +113,42 @@ def get_keys(pyg_data):
         return pyg_data.keys
     else:
         return pyg_data.keys()
+
+
+def found_size_mismatch(task: str, features: Union[Data, GraphDict], labels: np.ndarray, smiles: str):
+    """
+    Check if a size mismatch exists between features and labels with respect to node/edge/nodepair.
+   
+    """
+
+    mismatch = False
+
+    if task.startswith("graph_"):
+        pass
+
+    elif task.startswith("node_"):
+        if labels.shape[0] != features.num_nodes:
+            mismatch = True
+            logger.warning(
+                (f"Inconsistent number of nodes between labels and features for {smiles}: {labels.shape[0]} vs {features.num_nodes}")
```

Comment:
Add the `task` to the warning message

Reply:
The warning message has been updated here c7b68cd2810b7b6c0b9259f814e4f969b946232e